pack/eoe/eoep.json

```diff
@@ -1033,221 +1670,766 @@
+        "traits": "Talent.",
         "type_code": "asset",
-        "xp": 1
+        "xp": 3
     },
     {
-        "code": "08102",
-        "cost": 1,
+        "code": "08123",
+        "cost": 3,
         "deck_limit": 2,
         "faction_code": "seeker",
-        "faction2_code": "mystic",
-        "flavor": "All living things have breath.\nBe careful they do not curse you with theirs.",
-        "illustrator": "Chris Pritchard",
-        "name": "Protecting the Anirniq",
+        "faction2_code": "rogue",
+        "faction3_code": "survivor",
+        "illustrator": "Falk",
+        "name": "Crafty",
         "pack_code": "eoep",
-        "position": 102,
+        "position": 123,
         "quantity": 2,
-        "skill_agility": 1,
-        "skill_intellect": 1,
-        "text": "Fast. Play after an [[Ally]] asset at your location is discarded by a card effect or is defeated.\nEither return that asset to its owner's hand, or its owner draws 3 cards.",
-        "traits": "Ritual.",
-        "type_code": "event",
-        "xp": 2
-    },  
+        "skill_agility": 2,
+        "text": "Uses (2 resources). Replenish these resources at the start of each round.\nResources on Crafty may be spent to pay for [[Insight]], [[Tool]], or [[Trick]] cards.\n[fast] During a skill test on a [[Insight]], [[Tool]], or [[Trick]] card, spend 1 resource from Crafty: You get +1 skill value for this test.",
+        "traits": "Talent.",
+        "type_code": "asset",
+        "xp": 3
+    },
     {
-        "code": "08106",
+        "code": "08124",
         "cost": 3,
         "deck_limit": 2,
         "faction_code": "seeker",
-        "faction2_code": "survivor",
-        "health": 1,
-        "illustrator": "Romana Kendelic",
-        "is_unique": true,
-        "name": "Professor William Webb",
+        "faction2_code": "rogue",
+        "faction3_code": "mystic",
+        "illustrator": "Caravan Studio",
+        "name": "Antiquary",
         "pack_code": "eoep",
-        "position": 106,
+        "position": 124,
         "quantity": 2,
-        "sanity": 3,
-        "skill_intellect": 1,
-        "skill_wild": 1,
-        "slot": "Ally",
-        "subname": "Finder of Hidden Connections",
-        "text": "Uses (3 secrets).\n[reaction] When you successfully investigate, exhaust Professor William Webb and spend 1 secret: Choose an [[Item]] card in your discard pile and add it to your hand. Instead of discovering a clue at your location, you may discover a clue at a connecting location.",
-        "traits": "Ally. Miskatonic.",
+        "skill_agility": 2,
+        "text": "Uses (2 resources). Replenish these resources at the start of each round.\nResources on Antiquary may be spent to pay for [[Favor]], [[Relic]], or [[Ritual]] cards.\n[fast] During a skill test on a [[Favor]], [[Relic]], or [[Ritual]] card, spend 1 resource from Antiquary: You get +1 skill value for this test.",
```

Comment:
Annoyingly, there's no comma after [[Relic]] in the second line (even though that doesn't match the other 3-class talents). Please delete the one you've put in.

Reply:
Likewise after Relic for the fast action - please delete comma.

Reply:
Take 1 horror.